src/app/report/edit-form/edit-form.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, Input, OnChanges } from '@angular/core';
+import { Report } from '../../shared/interfaces/report';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+
+@Component({
+  selector: 'app-edit-form',
+  standalone: true,
+  imports: [ReactiveFormsModule, FormsModule],
+  templateUrl: './edit-form.component.html',
+  styleUrl: './edit-form.component.css',
+})
+export class EditFormComponent implements OnChanges {
+  @Input({ required: true }) report!: Report;
+  protected _name?: string;
+  protected _description?: string;
+  protected _path?: string;
+  protected _transformation?: string;
+  protected _variables?: string;
```

Comment:
Why start with prefixing variables with `_`?
Normally an `_` would indicate a private variable but that's not even the case here since you bind it in the template

Reply:
I just copied this code from another component, but it's because of the getters and setters

Reply:
But are you using the getters?
Not in the template at least, and if so maybe have template also use the getters so you can make all of these properties private (reflects my original comment)

Reply:
Well this whole component needs to be refactored, but this will be done later, this was just copied into it's own component to make the parent: edit-display.component more readable.